glean-core/ios/Glean/Net/HttpPingUploader.swift

```diff
@@ -0,0 +1,83 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+class HttpPingUploader {
+    let LOG_TAG = "glean/HttpPingUploader"
+
+    func logPing(path: String, data: String, config: Configuration) {
+        if config.logPings {
+            NSLog("\(LOG_TAG) : \(path)\n\(data)")
```

Comment:
I will never get used to `\(...)` for string interpolation...

Reply:
And I will never get used to `${}` a la Kotlin